hq/app/views/snyk/snyk.scala.html

```diff
@@ -0,0 +1,54 @@
+@import logic.SnykDisplay
+@import logic._
+@import model._
+
+@(results: List[SnykProjectIssues])(implicit assets: AssetsFinder)
+
+@main(List("Snyk")) { @* Header *@
+    <div class="container">
+        <div class="row">
+            <h1 class="header center-on-small-only grey-text text-lighten-5">Security HQ</h1>
+            <div class="col s12 m9">
+                <h4 class="grey-text text-darken-4 center-on-small-only">Snyk Results</h4>
+            </div>
+        </div>
+    </div>
+
+} { @* Main content *@
+
+    <div class="container">
+
+        <table class="striped responsive-table filterable-table">
+            <thead>
+                <tr>
+                    <th>Project Name</th>
+                    <th>High</th>
+                    <th>Medium</th>
+                    <th>Low</th>
+                </tr>
+            </thead>
+
+            <tbody>
+            @for(projectIssues <- results) {
+                <tr>
+                    <td>
+                        <a class="black-text" href="https://snyk.io/org/@projectIssues.project.get.organisation.get.name/project/@projectIssues.project.get.id/">@projectIssues.project.get.name</a>
+                    </td>
+                    <td>
+                        <a class="black-text" href="https://snyk.io/org/@projectIssues.project.get.organisation.get.name/project/@projectIssues.project.get.id/?disclosure=all&severity=high">@projectIssues.high</a>
+                    </td>
+                    <td>
+                        <a class="black-text" href="https://snyk.io/org/@projectIssues.project.get.organisation.get.name/project/@projectIssues.project.get.id/?disclosure=all&severity=medium">@projectIssues.medium</a>
+                    </td>
+                    <td>
+                        <a class="black-text" href="https://snyk.io/org/@projectIssues.project.get.organisation.get.name/project/@projectIssues.project.get.id/?disclosure=all&severity=low">@projectIssues.low</a>
+                    </td>
```

Comment:
Black text on cards and within nav is okay (and is the current pattern), since most users will expect those areas to be clickable. However, within a table or body text, links should be visually distinct (at least a different colour, ideally also with text decoration). https://github.com/guardian/security-hq/pull/64#discussion_r163819446

I feel that we should reuse the styling/design from the credentials report here:

<img width="1260" alt="picture 178" src="https://user-images.githubusercontent.com/8607683/35563966-6ee61442-05b0-11e8-8082-667b16483ab6.png">

Not a blocker - happy for this to be a change for a future PR